tools/pipelines/build-docs.yml

```diff
@@ -87,22 +90,54 @@ variables:
   - name: pnpmStorePath
     value: $(Pipeline.Workspace)/.pnpm-store
 
-# TODO: Move trigger to pipeline designated for artifact model generation. Currently,
-# the website docs builds and deploys on each commit. However, inconsistencies between
-# the release branches and main is causing the website to change unexpectedly we can
-# move the release/* trigger after separating the deploy step to another pipeline
-# no PR triggers
 trigger:
   branches:
     include:
-    - release/*
+    - release/**
```

Comment:
Looking at the current pipeline runs it seems like the release builds weren't triggering the build-docs pipeline. I assume it might be because the release branches are named 'release/client/2.4' and that the single * wild card might not cover it.